pkg/install/v1alpha1/version_test.go

```diff
@@ -40,3 +41,29 @@ func TestIsCurrentVersionValid(t *testing.T) {
 		})
 	}
 }
+
+func TestVersion(t *testing.T) {
+        tests := map[string]struct {
+                version    string
```

Comment:
this is also good but maybe `inputVersion` is better. what do you think @prateekpandey14 ?